src/foraging_gui/Foraging.py

```diff
@@ -109,6 +109,7 @@ def __init__(self, parent=None,box_number=1):
         self._LickSta()
         self._InitializeMotorStage()
         self._StageSerialNum()
+        self._warmup()
```

Comment:
The warmup period is separate from the actual session?

Reply:
The warmup is dependent on the actual session. The logic is when the warmup is turned on, we will use specific parameters for some fields (parameters in the `Warm up:`, `Advanced block:`, `Block:` , `Base reward probability:` and `Auto water:` ) and leave other parameters related to that session unchanged. When warmup is turned off, the parameters will return to the state before warmup was turned on.